docs/guide/docs/quickstart.md

```diff
@@ -3,16 +3,19 @@ id: quickstart
 title: Quick Start
 ---
 
-## Admin
+## Admin Dashboard
 
-The admin dashboard is the place where you'll be able to manage and configure everything related to your Botpress Server installation, including:
+### Features
 
-- Create new bots
-- Manage your Botpress Pro license and sync with version control
-- Update your profile and your Botpress Account
-- Unpack and activate modules
+Here you'll be able to manage and configure everything related to your Botpress Server installation, including:
 
-To open the Admin Dashboard, visit [`http://localhost:3000`](http://localhost:3000) in your favorite web browser. When you open the portal for the first time, you will be asked to register the main admin account
+- Creating new bots
+- Managing your Botpress Pro license and sync with version control
+- Updating your profile and your Botpress Account
+- Unpacking and activating modules
+
+To open the Admin Dashboard, visit [`http://localhost:3000`](http://localhost:3000) in your favorite web browser. When you open the portal for the first time, you'll be asked to register the main admin account
```

Comment:
There's one thing I always wanted to clarify but somehow never did. As of now, a user will never create a "Botpress account" A user will create an account on their installation. There is no such thing as a Botpress central authentication service (for now). 

I know that for some it's obvious but I know that for a lot of people this is not obvious at all. We should find a way to explain that they're only authenticating locally and not to a botpress hosted server. We should also differentiate that in the setup screen

Reply:
+1

Reply:
Added a note on that in the docs. As for the setup screen, not so sure how to ensure that it's eventually added by the interface design lot.